.travis.yml

```diff
@@ -108,11 +108,24 @@ matrix:
       stage: "CMake"
       name: "CMake blinky example"
       env: NAME=cmake_test
+      before_install:
+        #Install Cmake
+        - curl -L0 https://github.com/Kitware/CMake/releases/download/v3.18.0/cmake-3.18.0.tar.gz --output /tmp/cmake-3.18.0.tar.gz
```

Comment:
why do  we need to fetch and build cmake? we are using recommended pip wheels, that is up to date, see https://pypi.org/project/cmake/.

We could lock a version if we need to, if that was the goal here.

Reply:
@0xc0170 Upstream `cmake` cannot be used with Arm compiler shipped with Mbed Studio as it will result in an error. Therefore we need a patched version of `cmake`, hence build from source.
I'll add the patch soon.